db/migrations/20220818142407_add_unique_index_to_service_instance_operations_service_instance_id.rb

```diff
@@ -0,0 +1,35 @@
+Sequel.migration do
+  up do
+    # Remove duplicate service_instance_operations.service_instance_id to prepare for adding a uniqueness constraint
+    max_updated_at_group = self[:service_instance_operations].
```

Comment:
I find this hard to understand and thus difficult to verify. So I've pushed a commit with a test for correctly removing duplicates. When executing this test, it fails.

Maybe a different approach would be easier. We don't expect many duplicates, so first finding those service_instance_ids for which there are duplicates and then (for each of them) finding the duplicates that should be removed (keep latest or the one with the highest id) might be easier to implement and understand.